Modules/Core/Common/test/itkImageIORegionGTest.cxx

```diff
@@ -81,7 +81,7 @@ Expect_CopyAssignableToSelf(const T & value)
   T self(value);
 
   // Slight contortion to avoid self-assignment warning with 'self = self'.
-  const T& referenceToSelf = self;
+  const T & referenceToSelf = self;
```

Comment:
Personally I would prefer the original, to indicate more clearly that the ampersand is part of the type:

    T& reference = variable;

Whereas I like to see the ampersand at the right, when it means "address-of":

    ptr = &variable;

And in the middle on binary AND operation:

    result = byte1 & byte2; 

My two cents.... and best wishes for 2020! 🎉